tests/infra/ccf.py

```diff
@@ -105,7 +99,6 @@ def __init__(
                     host,
                     debug=str(local_node_id_) in (dbg_nodes or []),
                     perf=str(local_node_id_) in (perf_nodes or []),
-                    recovery=recovery,
                 )
 
     def start_and_join(self, args):
```

Comment:
Add `TODO` so that node 0 is not necessarily the node that starts (user should be able to pick).